encryption/BIKE/BIKE.yaml

```diff
@@ -0,0 +1,35 @@
+stateful: false
+name: BIKE
```

Comment:
can you add a `description`?

We can think of one together on slack if you don't have a good idea. I don't really have one :)

Reply:
I choose the standard description as used in the [introduction](https://bikesuite.org/files/v4.0/BIKE_Spec.2020.05.03.1.pdf#page=6&zoom=100,148,168) of the specification paper.